silkrpc/commands/eth_util_api.cpp

```diff
@@ -0,0 +1,69 @@
+/*
+    Copyright 2020 The Silkrpc Authors
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+#include <memory>
+#include <vector>
+
+#include <silkrpc/config.hpp> // NOLINT(build/include_order)
+
+#include <asio/awaitable.hpp>
+#include <asio/thread_pool.hpp>
+#include <evmc/evmc.hpp>
+#include <nlohmann/json.hpp>
+
+#include <silkrpc/context_pool.hpp>
+
+#include <silkworm/common/util.hpp>
+#include <silkworm/common/base.hpp>
+#include <silkworm/chain/config.hpp>
+#include <silkworm/common/log.hpp>
+
+#include <silkrpc/common/log.hpp>
+#include <silkrpc/types/receipt.hpp>
+
+#include <silkrpc/core/rawdb/accessors.hpp>
+#include <silkrpc/core/rawdb/chain.hpp>
+#include <silkrpc/types/block.hpp>
+#include <silkrpc/types/chain_config.hpp>
+
+#include <silkrpc/commands/eth_util_api.hpp>
```

Comment:
This implementation module should import the associated header instead of reincluding all the dependencies